fvtest/compilertriltest/ArithmeticTest.cpp

```diff
@@ -221,6 +221,10 @@ float fdiv(float l, float r) {
     return l/r;
 }
 
+float frem(float l, float r) {
+    return remainder(l, r);
```

Comment:
I believe `std::remainder()` is new in C++11. Are we sure it's available on all our supported build compilers?

Reply:
Thanks for the heads-up! I'll look for a different solution.

Reply:
@Leonardo2718 I just replaced it with remainderf(), which I believe is the C99 version.

Reply:
I'm not 100% sure yet that's going to be enough.

The C99 standard library is, technically, only "officially" available in C++ since C++11.

However, because most C++03 compilers ship with a C99 compiler as well, I suspect the C99 standard library *is* available pre-C++11 in most compilers.

Reply:
@Leonardo2718 Here is another solution done using floor() and ceil(), does this work?